pkg/ig-manager/ig-manager.go

```diff
@@ -98,7 +98,7 @@ func NewManager(runtimes []*containerutilsTypes.RuntimeConfig) (*IGManager, erro
 		containercollection.WithCgroupEnrichment(),
 		containercollection.WithLinuxNamespaceEnrichment(),
 		containercollection.WithMultipleContainerRuntimesEnrichment(runtimes),
-		containercollection.WithContainerFanotifyEbpf(),
+		containercollection.WithContainerFanotifyEbpf(false),
```

Comment:
Shouldn't this be false only if users pass `--host`? If such a flag is not passed, it means users want to debug only containers, so it doesn't make sense to go ahead.

Reply:
Some thoughts:
- It doesn't seem to easy to me getting the value of such parameter there. I see it's called from LocalManager.Init(), and there the operator params are available, but `--host` is not an operator but a gadget param IIUC.

https://github.com/inspektor-gadget/inspektor-gadget/blob/5a6dd27ae890d493929423eb452914c9bb76f867/pkg/operators/localmanager/localmanager.go#L163

- In the current implementation we could know the value of `--host`, but once https://github.com/inspektor-gadget/inspektor-gadget/issues/1681 is implemented we won't know that value at the time the daemon is started. It's even possible that the user tries to run different gadgets with and without --host
- I'm not totally convinced that the default should be `--host=false`, so I rather prefer not to tie those two things there.

Reply:
With the client/server approach, I agree it should go ahead. However, we need to provide some feedback when the user starts the server, the container-hook fails to get initialized, and they start a gadget without `--host`. Otherwise, container enrichment and filtering won't work, and they will have no idea what's happening because the error `Error: initializing operators: initializing operator "LocalManager": initializing manager: starting` will be only on the server side, when it was launched.

Maybe a new `gadgetctl status` could provide the server's status, including which hooks were installed and their statuses? It could also provide other info like the one for BTF: it was downloaded, or it's using the one from the system, etc. That could be useful to request when folks report an issue.